R/plotDataSetList.R

```diff
@@ -258,15 +260,18 @@ Plot.FV.ECDF_AUC <- function(dsList, rt_min = NULL, rt_max = NULL,
 #' @param scale.xlog Whether or not to scale the x-axis logaritmically
 #' @param scale.ylog Whether or not to scale the y-axis logaritmically
 #' @param algids Which algorithms from dsList to use
+#' @param par_name Which parameters to create plots for
+#' @param show.CI Whether or not to show the standard deviation
 #'
 #' @return A plot of for every recorded parameter in the DataSetList
 #' @export
 #' @examples 
 #' Plot.Parameters(subset(dsl, funcId == 1))
 Plot.Parameters <- function(dsList, f_min = NULL, f_max = NULL,
-                          algids = 'all',
+                          algids = 'all', par_name = NULL,
```

Comment:
par_name is set to NULL by default, meaning to plot all parameters found in dsList?

Reply:
Yes, this is true. I will add it to the description of the parameter